### ISSUE TITLE
Listener by-character and by-string output don't line up

### ISSUE THREAD

**Issue by jckarter** [factor/factor#496]
In the Mac UI with the default monospace font, the pipes output by 

``` factor
IN: scratchpad 20 [ CHAR: space write1 ] times CHAR: | write1 nl "                    |" print
```

should line up but don't.

**Comment by mrjbq7** [comment]
Better (almost fixed) by 824e239915b9420dd32c6b082ca6b7ec49ebe45c.

**Comment by mrjbq7** [comment]
Fixed completely by 68f6eeb3adbc7d9f71d24761600bf8538775c969.
